tests/kubernetes/para-single.json

```diff
@@ -63,7 +50,20 @@
         "args": [
           "-lparachain=debug"
         ]
-      }
+      },
+      "collator_groups": [
+        {
+          "name": "stps",
+          "ws_port": 9944,
+          "args": [
+            "-lparachain=debug",
+            "--no-hardware-benchmarks",
+            "--pool-limit=1000000000",
+            "--pool-kbytes=10000000000"
+          ],
+          "count": "{{SENDERS}}"
+        }
+      ]
```

Comment:
It seems the collator_groups are not started as expected when running `zombienet`.